docs/dev-tools/zombienet/index.md

```diff
@@ -0,0 +1,600 @@
+---
+title: Using Zombienet to spawn ephemeral substrate based networks
+description: Diving deeper into Zombienet, a versatile tool enabling the creation of temporary substrate networks for testing purposes
+---
+
+# Zombienet
+
+## Introduction
+
+Zombienet is a testing framework designed for Substrate-based blockchains. It provides a simple CLI tool for creating and testing blockchain environments locally or across networks. It supports various backend providers, including Kubernetes, Podman, and native setups for running blockchain nodes. 
+
+The framework enables developers to create tests using natural language tools, allowing them to verify on-chain storage, metrics, logs, and custom interactions with the blockchain. It is particularly effective for setting up local relaychains with validators and parachains with collators.
+
+This framework has been designed and developed by [Parity Technologies]({{ https://www.parity.io/ }}){target=_blanket}, now mantained by the Zombienet team. For further support and information, please refer to the following contact points:
+    
+- [Zombienet repository]( https://github.com/paritytech/zombienet ){target=_blanket}
+- [Element public channel]( https://matrix.to/#/!FWyuEyNvIFygLnWNMh:parity.io?via=parity.io&via=matrix.org&via=web3.foundation ){target=_blanket}
+
+## Installation
+
+
+Zombienet releases can be found on the [Zombienet repository]( https://github.com/paritytech/zombienet ){target=_blanket}. Each release includes executables for Linux and Macos, which are generated using [pkg]( https://github.com/vercel/pkg ){target=_blanket}. This allows the Zombienet CLI to operate without requiring Node.js to be installed. 
+
+In order to install Zombienet, there are some options available, depending on the user's preferences and the environment where it will be used. The following sections will guide you through the installation process for each of the available options.
```

Comment:
```suggestion
In order to install Zombienet, there are multiple options available, depending on the user's preferences and the environment where it will be used. The following sections will guide you through the installation process for each of the available options.
```